swagger.json

```diff
@@ -963,10 +963,7 @@
                 ],
                 "responses": {
                     "200": {
-                        "description": "Returns the reactivated driver, which is now available at /fleet/drivers/{driver_id}.",
-                        "schema": {
-                            "$ref": "#/definitions/CurrentDriver"
-                        }
+                        "description": "Successfully reactivated the driver, which is now referenced by /fleet/drivers/{driver_id | external_id}.",
```

Comment:
Should the PUT return the updated object back?

Reply:
The DELETE doesn't return updated object either, so I wanted to match that. Returning the updated object would also require some changes in the api.